src/storage/Manager.cc

```diff
@@ -23,7 +23,9 @@ void detail::ExpirationTimer::Dispatch(double t, bool is_expire) {
     // in the interim.
     if ( ! expire_running.test_and_set() ) {
         DBG_LOG(DBG_STORAGE, "Starting new expiration thread");
-        storage_mgr->expiration_thread = std::jthread([]() { storage_mgr->Expire(); });
+        double current_network_time = run_state::network_time;
```

Comment:
How about using `t` that is given to the timer's Dispatch() instead? I think they end up the same anyway right now.